spec/models/transformation_mapping_item_spec.rb

```diff
@@ -26,6 +26,7 @@
       expect(invalid_mapping_item.errors[:source].first).to match("EMS type of source cluster must be in")
     end
   end
+  # end of source cluster validation
```

Comment:
Is this comment necessary?

Reply:
IMO, it helps with reading and understanding the code.

Reply:
Agree with @bdunne here, if a conditional is so long that you lose context of what the start is then it is too long and should be a method

Reply:
It's also not consistent with the majority of the code base.

Reply:
Okay, I will remove them before the final submission.

Reply:
Done.